omod/src/main/java/org/openmrs/module/owa/web/controller/AddOnManagerRestController.java

```diff
@@ -0,0 +1,21 @@
+package org.openmrs.module.owa.web.controller;
+
+
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+
```

Comment:
Can you deprecate OwaRestController and also put a comment here that once OwaRestController is deleted, this class should be renamed to OwaRestController? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation